aten/src/ATen/core/interned_strings.h

```diff
@@ -326,6 +326,8 @@ namespace c10 {
   _(aten, special_erf)               \
   _(aten, erfc)                      \
   _(aten, special_erfc)              \
+  _(aten, erfcx)                     \
```

Comment:
This operator should reside just in `torch.special` namespace so we can get way with this.